src/Hl7.Fhir.Specification/Specification/Source/ResourceResolverExtensions.cs

```diff
@@ -47,7 +47,16 @@ public static StructureDefinition FindStructureDefinition(this IResourceResolver
 
         public static StructureDefinition FindStructureDefinitionForCoreType(this IResourceResolver resolver, string typename)
         {
-            var url = ModelInfo.CanonicalUriForFhirCoreType(typename);
+            string url;
```

Comment:
I would prefer the following style:
`var url = Uri.IsWellFormedUriString(typename, UriKind.Absolute) ? typename : ModelInfo.CanonicalUriForFhirCoreType(typename)`

But technically it is exact the same, so you can decide what to do with that..

Reply:
Changed as requested